### ISSUE TITLE
Automatic reset of external validation errors

### ISSUE THREAD

**Issue by entermix** [vuelidate/vuelidate#891]
**Is your feature request related to a problem? Please describe.**

```
// externalErrors ... 
const validator = useVuelidate(
      rules,
      state,
      {
        $autoDirty: true,
        $externalResults: externalErrors,
      }
    );
```

With this configuration, if the field is changed by the user, errors do not disappear automatically.

To clear external errors, we must call the `$clearExternalResults()` method

How about adding the ability to automatically clean up errors after changing a field if the `$autoDirty` option is enabled?

What do you think about it?

**Additional context**
https://github.com/vuelidate/vuelidate/issues/824
https://github.com/vuelidate/vuelidate/pull/837

**Comment by dobromir-hristov** [comment]
That is harder to do on a per-item basis... but I will think about it, thanks!

**Comment by dobromir-hristov** [comment]
#916 Should fix this for you :)

**Comment by entermix** [comment]
Thank you! Will this feature be available in version 2.0.0-alpha.24?
